scico/optimize/pgm.py

```diff
@@ -387,8 +387,7 @@ class PGM:
 
     Minimize a function of the form :math:`f(\mb{x}) + g(\mb{x})`.
 
-    The function :math:`f` must be smooth and :math:`g` must have a
-    defined prox.
+    The function :math:`g` must have a defined prox.
```

Comment:
It's worth mentioning that f should be smooth, but that the algorithm will work for f with points at which it's non-smooth as long as those points are not visited during the optimization.

Reply:
Will at also work if valid subgradients are returned for nonsmooth points? It would be valuable to say so (with a ref) if it is true.

Reply:
Perhaps @crstngc can comment.

Reply:
> It's worth mentioning that f should be smooth, but that the algorithm will work for f with points at which it's non-smooth as long as those points are not visited during the optimization.

Generally agree, however,

> as long as those points are not visited

Does *as long as* mean the same as *if and only if*?

Reply:
Let's say

_guaranteed to work if globally smooth_

(leaving it open to the interpretation of the user what happens with not globally smooth losses)

Reply:
See 02fe9bb